04_HowTos/DataReduction/DRAGONS_reduction_examples/GSAOI_Imaging_EllipticalGalaxy/GSAOI_Imaging_EllipticalGalaxy.ipynb

```diff
@@ -350,7 +354,7 @@
     "\n",
     "The sky frame comes from off-target sky observations. We feed the pipeline all the on-target and off-target frames. The software will appropriately split the on-target and off-target using information in the headers.\n",
     "\n",
-    "This command will generate flat-corrected and sky-subtracted files but will not stack them. You can differentiate each file by its suffix (_flatCorrected or _skyCorrected). The on-target files are the ones that have been sky subtracted (_skyCorrected). There should be nine of them.\n",
+    "This command will generate flat-corrected and sky-subtracted files but will not stack them. You can differentiate each file by its suffix (_flatCorrected or _skyCorrected). The on-target files are the ones that have been sky subtracted (_skyCorrected). Finally, the stacked image will be saved as S20170505S0095_image.fits.\n",
```

Comment:
Assuming there won't be other images before with the suffix _image.fits, change
_"the stacked image will be saved as S20170505S0095\_image.fits."_
with
_"the stacked image will be saved with the suffix \_image.fits."_